secrets.env.sample

```diff
@@ -0,0 +1,41 @@
+OBJECT1_NAME={{OBJECT1_NAME}}
+OBJECT1_VALUE={{OBJECT1_VALUE}}
+OBJECT1_TYPE={{OBJECT1_TYPE}}
+OBJECT1_ALIAS={{OBJECT1_ALIAS}}
+OBJECT1_VERSION={{OBJECT1_VERSION}}
+
+OBJECT2_NAME={{OBJECT2_NAME}}
+OBJECT2_VALUE={{OBJECT2_VALUE}}
+OBJECT2_TYPE={{OBJECT2_TYPE}}
+OBJECT2_ALIAS={{OBJECT2_ALIAS}}
+OBJECT2_VERSION={{OBJECT2_VERSION}}
+
+CERT1_NAME={{CERT1_NAME}}
+CERT1_VERSION={{CERT1_VERSION}}
+CERT1_VALUE={{CERT1_VALUE}}
+CERT1_SECRET_VALUE={{CERT1_SECRET_VALUE}}
+CERT1_KEY_VALUE={{CERT1_KEY_VALUE}}
+
+CERT2_NAME={{CERT2_NAME}}
+CERT2_VERSION={{CERT2_VERSION}}
+CERT2_VALUE={{CERT2_VALUE}}
+CERT2_SECRET_VALUE={{CERT2_SECRET_VALUE}}
+CERT2_KEY_VALUE={{CERT2_KEY_VALUE}}
+
+CERT3_NAME={{CERT3_NAME}}
+CERT3_VERSION={{CERT3_VERSION}}
+CERT3_VALUE={{CERT3_VALUE}}
+CERT3_SECRET_VALUE={{CERT3_SECRET_VALUE}}
+CERT3_KEY_VALUE={{CERT3_KEY_VALUE}}
+
+KEYVAULT_NAME={{KEYVAULT_NAME}}
+AZURE_CLIENT_ID={{AZURE_CLIENT_ID}}
+AZURE_CLIENT_SECRET={{AZURE_CLIENT_SECRET}}
+TENANT_ID={{TENANT_ID}}
+
+DOCKER_IMAGE={{DOCKER_IMAGE}}
+IMAGE_TAG=0.0.1
```

Comment:
Not a blocker for this PR, but I think we should consider using just `IMAGE_VERSION` and remove `IMAGE_TAG`